core/modules/dashboard/includes/block.news.inc

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * @file
+ * Dashboard block displaying BackdropCMS.org news items.
+ */
+
+class DashboardNewsBlock extends Block {
+  /**
+   * {@inheritdoc}
+   */
+
+  public function __construct($plugin_name, array $data) {
```

Comment:
If there is no new code, then the parent method is already inherited. So you dont need the constructor at all.

Reply:
Deleted